apps/backend/src/handlers/command-handlers.ts

```diff
@@ -0,0 +1,156 @@
+  pool: Pool
+  commandRegistry: CommandRegistry
+  streamService: StreamService
+  jobQueue: JobQueueManager
+}
+
+export function createCommandHandlers({ pool, commandRegistry, streamService, jobQueue }: Dependencies) {
+  return {
+    /**
+     * Dispatch a slash command.
+     *
+     * This validates the command, creates a command_dispatched event,
+     * queues the command for execution, and returns an ack.
+     */
+    async dispatch(req: Request, res: Response) {
+      const userId = req.userId!
+      const workspaceId = req.workspaceId!
+
+      const result = dispatchCommandSchema.safeParse(req.body)
+      if (!result.success) {
+        return res.status(400).json({
+          success: false,
+          error: "Validation failed",
+          details: z.flattenError(result.error).fieldErrors,
+        })
+      }
+
+      const { command: commandString, streamId } = result.data
+
+      // Validate stream access
+      const [stream, isStreamMember] = await Promise.all([
+        streamService.getStreamById(streamId),
+        streamService.isMember(streamId, userId),
+      ])
+
+      if (!stream || stream.workspaceId !== workspaceId) {
+        return res.status(404).json({
+          success: false,
+          error: "Stream not found",
+        })
+      }
+
+      if (!isStreamMember) {
+        return res.status(403).json({
+          success: false,
+          error: "Not a member of this stream",
+        })
+      }
+
+      // Parse the command
+      const parsed = parseCommand(commandString)
+      if (!parsed) {
+        return res.status(400).json({
+          success: false,
+          error: "Invalid command format. Commands must start with / followed by a name.",
+        })
+      }
+
+      // Look up the command
+      const cmd = commandRegistry.get(parsed.name)
+      if (!cmd) {
+        return res.status(404).json({
+          success: false,
+          error: `Unknown command: ${parsed.name}`,
+          availableCommands: commandRegistry.getCommandNames(),
+        })
+      }
+
+      // Create command_dispatched event
+      const cmdId = generateCommandId()
+      const evtId = eventId()
+
+      const event = await withTransaction(pool, async (client) => {
+        const evt = await StreamEventRepository.insert(client, {
+          id: evtId,
+          streamId,
+          eventType: "command_dispatched",
+          payload: {
+            commandId: cmdId,
+            name: parsed.name,
+            args: parsed.args,
+            status: "dispatched",
+          } satisfies CommandDispatchedPayload,
+          actorId: userId,
+          actorType: "user",
+        })
+
+        // Publish to outbox for author-only broadcast
+        await OutboxRepository.insert(client, "command:dispatched", {
+          workspaceId,
+          streamId,
+          event: serializeBigInt(evt),
+          authorId: userId,
+        })
+
+        return evt
+      })
+
+      // Queue the command for execution
```

Comment:
This isn't durable enough, I'm sorry. This should be handled on `command:dispatched`, not outside the transaction in the handler.